test/engine/SpatialJoinPrefilterTest.cpp

```diff
@@ -670,47 +671,52 @@ TEST(SpatialJoinTest, BoundingBoxPrefilterDeactivatedTooLargeBox) {
   auto vIdUni = getValId(nMap, "uni");
   auto vIdGkAllee = getValId(nMap, "gk-allee");
 
-  // Intersects with prefiltering, but prefiltering is not used due to too large
-  // bounding box
-  SweeperTestResult testResult;
-  runParsingAndSweeper(qec, "other", "uni-separate", {INTERSECTS}, testResult,
-                       true, true);
-  checkSweeperTestResult(vMap, testResult,
-                         {{{INTERSECTS, vIdUni, vIdUniSep, 0},
-                           {INTERSECTS, vIdGkAllee, vIdUniSep, 0}},
-                          boundingBoxVeryLarge,
-                          boundingBoxUniSeparate,
-                          8,
-                          0,
-                          7,
-                          1},
-                         INTERSECTS, true);
+  {
+    [[maybe_unused]] const auto& runtimeParameter =
```

Comment:
Do you need the `[[maybe_unused]]`? I'm pretty sure most usages just do `auto cleanup = ...` without const reference and are just fine